### DISCUSSION TITLE
How to export environment variables from one app to another app

### DISCUSSION THREAD

**Discussion by Kutbi-dev** [codemagic-ci-cd/codemagic-docs#2904] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

Earlier we have 2 different repositories to manage two different project and build differently via codemagic pipeline.
Now, we have merged these two repo into one and also there are two different app created in code magic using two separate repo url to build pipeline.
Now we have only one repo url and published two different apps from one application defined in the codemagic.
My question is, the other app which we are not going to use in codemagic have the environment variables define in it.
How can I import those into another app from which we will build the pipeline in codemagic? 

### Expected results

I wanted to export environment variable settings of one app to another app.

### Actual results

It is not allowed me to export environment variables.

**Comment by dam-ease** [comment, marked as answer]
Hello @Kutbi-dev, 
I'm afraid Codemagic currently does not support exporting and importing environment variables directly from one app to another.
From your use case, it seems that updating the repository URL in your existing app i.e. the one with the configuration files/environment variables (let's refer to it as the "old app") might be a suitable solution if you'd like to keep all configurations in one place.

To achieve this:
1. Retain the "old app" with its current configuration.
2. We update the repository URL in the app settings to point to the new repository.
This way, you can reuse the environment variables and workflows already set up in the "old app," without needing to create a separate configuration for the new repository.

Let me know if you think this will suit your use-case and I'd be glad to help with the move.
